Resources/Audio/Machines/attributions.yml

```diff
@@ -37,3 +37,8 @@
   license: "CC0-1.0"
   copyright: "Modified from https://freesound.org/people/Pagey1969/sounds/566048/"
   source: "https://github.com/space-wizards/ss14-raw-assets/tree/99fcffb02d9953f031991fc8f9980f0c3abd3803/Audio"
+  
+- files: ["warning_buzzer.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Taken from TG station."
```

Comment:
TG isn't NC

Reply:
also the commit...

Reply:
ok, I changed this one to something off of freesound but I'll keep that in mind

Reply:
you didn't need to change the sound. The only thing you had to change was the license to `CC-BY-SA-3.0`.

Reply:
Oh I didn't even notice that detail